### PR TITLE
docs: update cluster create and change to ALPHA

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,25 +67,72 @@ spec:
...
+- **targetSize**: the number or percent of pods that can or cannot be placed on the node.
 - **weight**: allows the engine to decide which policy to use when pods match multiple policies.
 
 ### Demo
 
 - Create a [KinD](https://kind.sigs.k8s.io/) cluster with the following config
 
 ```sh
-kind create cluster --config https://raw.githubusercontent.com/Azure/placement-policy-scheduler-plugins/main/test/e2e/kind-config.yaml
+cat <<EOF | kind create cluster --name placement-policy --config=-
```
Comment (aramase) [inline README.md:86]: Changing this to line because `kind create cluster` doesn't work with remote urls

**Review by helayoty — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 - A scorer plugin implemented with that will be used in case “best effort” policy enforcement.
   - Extension points implemented: [PreScore](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#pre-score) and [Score](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#score)
 - A filter plugin that will be used in case “force” policy enforcement.
   - Extension points implemented: [PreFilter](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#pre-filter) and [Filter](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#filter)
 
-**WARNING:** This is experimental code. It is not considered production-grade by its developers, nor is it "supported" software.
+**WARNING:** This code is **ALPHA**. It is not considered production-grade by its developers, nor is it "supported" software.
```
Comment (khenidak) [inline README.md:12]: Note not warning.

Comment (khenidak) [reply README.md:12]: alpha stage. Not ready yet. remove the support note.

**Review by helayoty — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aramase) [reply README.md]: Thanks, fixed!

**Pull request merged**